.github/workflows/s3-tests.yml

```diff
@@ -0,0 +1,318 @@
+          sudo chmod a+w vendor/hosts
+        working-directory: neofs-dev-env
+
+      - name: Prepare Dev-Env to run tests
+        id: prepare_test_env
+        timeout-minutes: 30
+        run: |
+          make prepare-test-env
+          echo "$(pwd)/vendor" >> $GITHUB_PATH
+        working-directory: neofs-dev-env
+
+      - name: Set zero fee
+        run: |
+          make update.container_fee val=0 && make update.container_alias_fee val=0
+        working-directory: neofs-dev-env
+
+      - name: Copy neofs-s3-gw binaries to vendor directory
+        run: |
+          cp bin/* ${GITHUB_WORKSPACE}/neofs-dev-env/vendor/
+        working-directory: neofs-s3-gw
+
+################################################################
+
+      - name: Log environment
+        run: |
+          echo "Check free space"
+          df -h
+          echo "=========================================="
+
+          echo "Check /etc/hosts"
+          cat /etc/hosts
+          echo "=========================================="
+
+          echo "Check docker images"
+          docker images
+          echo "=========================================="
+
+          echo "Check docker ps"
+          docker ps
+          echo "=========================================="
+
+          echo "Check neo-go version"
+          neo-go --version
+          echo "=========================================="
+
+          echo "Check neofs-s3-authmate version"
+          neofs-s3-authmate --version
+          echo "=========================================="
+
+          echo "Check neofs-s3-gw version"
+          echo "=========================================="
+          neofs-s3-gw --version
+          echo "=========================================="
+
+          echo "Check neofs-adm version"
+          neofs-adm --version
+          echo "=========================================="
+
+          echo "Check neofs-cli version"
+          neofs-cli --version
+          echo "=========================================="
+
+          echo "Check vendor dir"
+          ls -lah "${GITHUB_WORKSPACE}/neofs-dev-env/vendor"
+          echo "=========================================="
+
+        working-directory: neofs-dev-env
+
+################################################################
+
+      - name: Copy test-env file to .env for neofs-s3-dev-env
+        run: |
+          cp .env ${GITHUB_WORKSPACE}/neofs-s3-dev-env/.env
+        working-directory: neofs-dev-env
+
+      - name: Copy neofs-s3-authmate for neofs-s3-dev-env
+        run: |
+          cp ${GITHUB_WORKSPACE}/neofs-dev-env/vendor/neofs-s3-authmate bin/s3-authmate
+        working-directory: neofs-s3-dev-env
+
+      - name: Prepare test S3-gw test env
+        run: |
+          make prepare.s3-gw
+        working-directory: neofs-s3-dev-env
+
+      - name: Copy s3tests config to s3-tests directory
+        run: |
+          cp services/s3-gw/s3tests.conf ${GITHUB_WORKSPACE}/s3-tests/${S3_TESTS_CONFIG}
```

Comment:
But there's no such file:
![image](https://github.com/nspcc-dev/neofs-s3-gw/assets/126654550/9195a7c6-e3ac-438b-a2f7-5a8992ce26b7)

Reply:
services/s3-gw/s3tests.conf is a result file of running 

`make prepare.s3-gw `
for s3-dev-env on the previous step

<img width="540" alt="image" src="https://github.com/nspcc-dev/neofs-s3-gw/assets/13661886/3d72af44-b0af-45ea-a33a-23728955c7f6">